src/index.html

```diff
@@ -9,10 +9,135 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--mon calendar--month-length--31">
+      <div
+        class="calendar__day"
+        data-day="1"
+      ></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
```
you don't need any data-attribute